doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1039 @@
+
+    * The way buffers are declared is updated when the instance is created.
+      Now the memory-region devicetree property works independently for each instance and does not require the specific instance name.
+
+* :ref:`caf_sensor_manager`:
+
+  * Updated by cleaning up :file:`sensor_event.h` and :file:`sensor_manager.h` files.
+    Moved unrelated declarations to a separate :file:`caf_sensor_common.h` file.
+
+DFU libraries
+-------------
+
+* :ref:`lib_dfu_target` library:
+
+  * Added:
+
+    * :kconfig:option:`CONFIG_DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION` configuration option to support the ``FMFU_STORAGE`` partition in external flash.
+    * :c:func:`dfu_target_full_modem_fdev_get` function that gets the configured flash device information.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`west_sbom`:
+
+  * Updated so that the output contains source repository and version information for each file.
+
+* :ref:`partition_manager`:
+
+  * Added:
+
+    * The :file:`ncs/nrf/subsys/partition_manager/pm.yml.fmfu` file.
+    * Support for the full modem FOTA update (FMFU) partition: ``FMFU_STORAGE``.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``cfec947e0f8be686d02c73104a3b1ad0b5dcf1e6``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added an option to prevent inclusion of default nRF5340 network core DFU image hook, which allows a custom implementation by users if the :kconfig:option:`CONFIG_BOOT_IMAGE_ACCESS_HOOK_NRF5340` Kconfig option is disabled (enabled by default).
+  CMake can be used to add additional hook files.
+  See :file:`modules/mcuboot/hooks/CMakeLists.txt` for an example of how to achieve this.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``e1e06d05fa8d1b6ac1b0dffb1712e94e308861f8``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline cd16a8388f ^71ef669ea4
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^cd16a8388f
+
+The current |NCS| main branch is based on revision ``cd16a8388f`` of Zephyr.
```

Comment:
@tmon-nordic , kindly please verify the final hashes for the release and provide the updated git log commands.

Reply:
`git log --oneline e1e06d05fa ^cd16a8388f`

`git log --oneline manifest-rev ^e1e06d05fa`

Based on revision ``e1e06d05fa`` (it is just the first 10 characters from the full hash above).